TSPL.docc/LanguageGuide/Concurrency.md

```diff
@@ -929,12 +925,18 @@ an *unstructured task* doesn't have a parent task.
 You have complete flexibility to manage unstructured tasks
 in whatever way your program needs,
 but you're also completely responsible for their correctness.
-To create an unstructured task that runs on the current actor,
+
+To create an unstructured task
+that runs similarly to the surrounding code,
 call the [`Task.init(priority:operation:)`][] initializer.
-To create an unstructured task that's not part of the current actor,
-<!-- XXX Using "actor" before defining it -->
+The new task defaults to running with
+the same actor isolation, priority, and task-local state as the current task.
+To create an unstructured task
+that's more independent from the surrounding code,
 known more specifically as a *detached task*,
-call the [`Task.detached(priority:operation:)`][] class method.
+call the [`Task.detached(priority:operation:)`][] static method.
+The new task defaults to running as a nonisolated function
```

Comment:
In the SE-0461 world, saying it runs as a nonisolated function has no implications about where it runs. Do we want to say it runs as an `@concurrent` function, or some other description of running off of an actor by default?